pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/performance/xml/InefficientStringBuffering.xml

```diff
@@ -301,7 +301,7 @@ public class Foo {
 
     <test-code>
         <description>1503099, append with two string lengths</description>
-        <expected-problems>1</expected-problems>
+        <expected-problems>0</expected-problems>
```

Comment:
Now it is correct :smile: For reference, that was the old bug: https://sourceforge.net/p/pmd/bugs/522/
But this indicates, that the bug was actually never really fixed... :upside_down_face: